dCommon/dClient/AssetManager.h

```diff
@@ -54,6 +54,7 @@ struct AssetStream : std::istream {
 	}
 
 	operator bool() {
+		// NEED TO FIX THIS
```

Comment:
if this is changed make sure to check files existing and not existing with both packed and unpacked clients